scripts/install-template.js

```diff
@@ -0,0 +1,102 @@
+const fs = require('fs');
+const path = require('path');
+const shelljs = require('shelljs');
+
+const matchingPackages = /^@commercetools-frontend\/(?!ui-kit)(.*)/;
+const applicationName = 'my-starter-app';
+const rootPath = path.join(__dirname, '..');
+const tarballsDistPath = path.join(rootPath, 'dist-tarballs');
+const sandboxPath = path.join(rootPath, '..', 'sandbox-installation-template');
+const binaryPath = path.join(sandboxPath, 'node_modules/.bin/create-mc-app');
+const applicationPath = path.join(sandboxPath, applicationName);
+
+const templateName = process.env.TEMPLATE_NAME;
+if (!templateName) {
+  throw new Error('Missing required environment variable "TEMPLATE_NAME"');
+}
+
+if (!fs.existsSync(tarballsDistPath)) {
+  throw new Error(
+    'The dist-tarballs folder does not exist. Please run the build-tarballs.js script before this.'
+  );
+}
+
+const branchName =
+  process.env.GITHUB_EVENT_NAME === 'pull_request'
+    ? process.env.GITHUB_HEAD_REF
+    : 'master';
+
+console.log('Cleaning up sandbox folder');
+shelljs.rm('-rf', sandboxPath);
+shelljs.mkdir('-p', sandboxPath);
+
+const createMcAppTarballName = shelljs
+  .ls(tarballsDistPath)
+  .find(tarballName => tarballName.includes('create-mc-app'));
+if (!createMcAppTarballName) {
+  throw new Error('Failed to find tarball for create-mc-app package');
+}
+
+console.log(`Installing create-mc-app from tarball`);
```

Comment:
We install this first, so that we can install the template using the binary script